_data/simple-icons.json

```diff
@@ -1409,11 +1409,6 @@
             "hex": "DC0D15",
             "source": "https://www.cdprojekt.com/en/media/logotypes/"
         },
-        {
-            "title": "Celery",
-            "hex": "37814A",
-            "source": "http://www.celeryproject.org/"
```

Comment:
The docs is still available:
https://docs.celeryproject.org/en/stable/

Reply:
Was about to say the same. Their Alexa rank is still within our scope and [their primary repo](https://github.com/celery/celery/) is still very much active, too.

Reply:
Just spotted that. Went by the quick glance at the website - as that's all the info we have in the JSON. Will remove that one now.

Reply:
The source will need to be updated, for sure :)

Reply:
Resolved in #5273, didn't want to mix updates / deletions in same PR.